DeAceroDLabs/marketplace-frontend
# Feature/dmp 525 Check products (step 2)  on payment view.

## What is done

[[DMP-525] Check products on payment view](https://deacero.atlassian.net/browse/DMP-525)

## How is done

- Created a `PaymentItem` component that will be used to show all the items (products) on the Payment view.
- Added margin prop to `Divider` component, cause sometimes we will want the divider to have margin, but sometimes not.
- Added action prop `Section` component that will be used on the Payment view to re-use it on every payment step when a button is needed beside title.
- Added variant prop to `Card` component  to use "no-shadow" | "grey-card" classes;  

## How to test

1. Run the project via `npm start`.
2. login.
3. Go to `carrito/pago`. You should see the Check products step.
